src/utils/apis.ts

```diff
@@ -370,3 +373,61 @@ export const openPort = async (port: number) => {
     })
   );
 };
+
+/***********************
+ * Start Tasks/Macro API
+ ***********************/
+
+export const getTasks = async (
```

Comment:
I think it might be better to remove the queryClient from these.
We'll probably pass these functions into `setQuery` and do it that way.